### ISSUE TITLE
Remove duplicate requests on /discussion page loading

### ISSUE THREAD

**Issue by jnaviask** [hicommonwealth/commonwealth#3615]
## Description
May be related to #3598 and #3599.

Currently, the discussion page loads the following routes when logged out:

```
1. /domain (check custom domain)
2. /status (load main site data)
3. /entities
4. /getEntityMeta
5. /bulkOffchain
6. /activeThreads (threads per topic = 3 => overview page)
7. /threadsUsersCountAndAvatars
8. /bulkThreads
9. /entities (dupe?)
10. /bulkThreads (dupe?)
11. /entities (dupe?)
12. /reactionCounts
13. /threadsUsersCountAndAvatars (dupe?)
14. /reactionCounts (dupe?)
15. /threadsUsersCountAndAvatars (dupe?)
16. /getAddressProfile (happens sometime after step 7, may be several)
```

Note that calls 9, 10, 11, 13, 14, and 15 are duplicates of prior queries. There is no reason for us to be making duplicate queries, as we have already fetched the data required.

Note that the overview page does not suffer from the same duplication of initialization calls.

## Acceptance Criteria
- The /discussions page should not make duplicate queries for the same data.

**Comment by kurtisassad** [comment]
We don't have a "already fixed" tag, so I have marked as a duplicate, as it has been fixed by @mzparacha
